spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -119,6 +125,27 @@ object DynamicConformanceJob {
     }
   }
 
+  private def getPostProcessingService(cmd: ConfCmdConfig, pathCfg: PathCfg, reportVersion: Int,
+                                       runNumber: Option[Int], uniqueRunId: Option[String]
+                                      )(implicit fsUtils: FileSystemVersionUtils): PostProcessingService = {
+    val runId = MenasPlugin.runNumber
+
+    if (runId.isEmpty) {
+      log.warn("No run number found, the Run URL cannot be properly reported!")
+    }
+
+    // reporting the UI url(s) - if more than one, its comma-separated
+    val runUrl: Option[String] = runId.map { runNumber =>
+      menasBaseUrls.map { menasBaseUrl =>
+        MenasRunUrl.getMenasUiRunUrl(menasBaseUrl, cmd.datasetName, cmd.datasetVersion, runNumber)
+      }.mkString(",")
```

Comment:
Is this assumption acceptable: reporting UI URL and if there are more, making them comma separated? The feature description did not dive into details what is expected in this regard...

Reply:
Yeah, I think this is good. Alternatively, we can include just 1 URL, the first one. Or the current active one.

Requirements might get polished when they start consuming and using the messages.